contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/MarkdownEditor/MarkdownEditor.vue

```diff
@@ -492,7 +530,7 @@
         }
 
         // open formulas menu if a math field clicked
-        const formulasMenuFormula = this.mathQuill(mathFieldEl).latex();
+        const formulasMenuFormula = mathFieldEl.getVueInstance().latex;
```

Comment:
`getVueInstance` is the method for accessing a `vue-custom-element` web component's underlying Vue component